src/plugins/cft/constants/transactions.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @author Ryadnov Andrey <me@ryadnov.ru>
+ */
+
+/**
+ * @type {{CASH: string, DEPOSIT_PROC: string, FDT_RBS_COMMISSION: string, PAYMENT: string, PURCHASE: string, RECHARGE: string, TRANSFER: string, WALLET: string, ZKDP: string}}
+ */
+const TYPES = {
+    CASH:               'CASH',
```

Comment:
does it really make any sense besides boosting the amount of code in dynamic environment?
you can still get `undefined` on typo :)